src/include/sof/math/exp_fcn.h

```diff
@@ -26,6 +26,38 @@
 
 #endif
 
-int32_t sofm_exp_int32(int32_t x);
+/* TODO: Is there a MCPS difference */
+#define USING_QCONVERT	1
```

Comment:
I already approved, but let's check in real devices if setting this to zero speeds up the code. In theory it shouldn't impact since Q_CONVERT_FLOAT macro should be evaluated in C pre-processor. Once confirmed we can remove the direct integers. Or if difference seen, remove the Q_CONVERT_FLOAT part.

Reply:
ok, can we do this as next steps in a follow up PR and test drive this now.